pkg/common/util/reconciler.go

```diff
@@ -15,17 +15,10 @@
 package util
 
 import (
-	"fmt"
-	"reflect"
-
-	corev1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"sigs.k8s.io/controller-runtime/pkg/event"
-
 	kubeflowv1 "github.com/kubeflow/training-operator/pkg/apis/kubeflow.org/v1"
 	"github.com/kubeflow/training-operator/pkg/controller.v1/common"
 	"github.com/kubeflow/training-operator/pkg/controller.v1/expectation"
-	commonutil "github.com/kubeflow/training-operator/pkg/util"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Could you make dependencies as group like

```
<Go std libs>

<Third party libs>

<ourselves libs>
```

Reply:
I've updated it. I've also folded the two `reconciler.go` and `reconciler_generic.go` files now that it's generics all along.

Reply:
Thanks!